scripts/extrinsic_check.sh

```diff
@@ -38,7 +38,7 @@ for RUNTIME in "${runtimes[@]}"; do
   )
```

Comment:
@nathanwhit FTR I didn't change line 36 b/c the latest git tag will still have the version in `src/lib.rs`.

Once we make the next release this should fail and we should update it then.